python-sdk/tests/databases/test_base_database.py

```diff
@@ -18,6 +18,24 @@ class DatabaseSubclass(BaseDatabase):
     pass
 
 
+def test_openlineage_database_dataset_namespace():
```

Comment:
just wondering does this test add some value?

Reply:
It is just for code coverage. It was failing for coverage

Reply:
I think this is a good point from @pankajastro, the test should check that the method returns what you expect. 
Test coverage reports tell you where you're missing tests, coverage is not a goal in itself.
In this case, the test should check that the method returns a namespace that conforms with the spec.

Reply:
Sure. We already have a test which covers the scenarios: https://github.com/astronomer/astro-sdk/blob/main/python-sdk/tests/sql/test_table.py#L221